### DISCUSSION TITLE
Operation has completed with phase: Error

### DISCUSSION THREAD

**Discussion by 13567436138** [argoproj/argo-cd#6644] [Q&A]
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [ ] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [ ] I've included steps to reproduce the bug.
* [ ] I've pasted the output of `argocd version`.

**Describe the bug**
Message:            ComparisonError: failed to sync cluster https://10.68.0.1:443: failed to load initial state of resource AuthorizationPolicy.security.istio.io: authorizationpolicies.security.istio.io is forbidden: User "system:serviceaccount:argocd-system:argocd-application-controller" cannot list resource "authorizationpolicies" in API group "security.istio.io" at the cluster scope;ComparisonError: failed to sync cluster https://10.68.0.1:443: failed to load initial state of resource AuthorizationPolicy.security.istio.io: authorizationpolicies.security.istio.io is forbidden: User "system:serviceaccount:argocd-system:argocd-application-controller" cannot list resource "authorizationpolicies" in API group "security.istio.io" at the cluster scope
FATA[0001] Operation has completed with phase: Error

**To Reproduce**

A list of the steps required to reproduce the issue. Best of all, give us the URL to a repository that exhibits this issue.
 argocd app sync guestbook
**Expected behavior**
sync ok

**Screenshots**

**Version**

```shell
Paste the output from `argocd version` here.
```

**Logs**

```
Paste any relevant application logs here.
```

**Comment by jannfis** [comment]
It seems the `ServiceAccount` used by your `argocd-application-controller` was not given the proper RBAC permissions required to run. The default cluster scoped installation manifests create these RBAC roles for you. Have you used another installation method?
